GenderPayGap.WebUI/Views/ActionHubOld/Effective.cshtml

```diff
@@ -0,0 +1,200 @@
+                If women negotiate their salaries more, they will end up with salaries that more closely match the salaries of men.
+            </p>
+        </li>
+        
+        <li>
+            <h3 class="govuk-heading-s">
+                Introduce transparency to promotion, pay and reward processes
+            </h3>
+            <p class="govuk-body">
+                Transparency means being open about processes, policies and criteria for decision-making.
+                This means employees are clear what is involved, and that managers understand that their decisions need to be objective and evidence-based because those decisions can be reviewed by others.
+                Introducing transparency to promotion, pay and reward processes can reduce pay inequalities.
+            </p>
+        </li>
+        
+        <li>
+            <h3 class="govuk-heading-s">
+                Appoint diversity managers and/or diversity task forces
+            </h3>
+            <div>
+                <p class="govuk-body">
+                    Diversity managers and task forces monitor talent management processes (such as recruitment or promotions) and diversity within the organisation.
+                    They can reduce biased decisions in recruitment and promotion because people who make decisions know that their decision may be reviewed.
+                    This accountability can improve the representation of women in your organisation.
+                </p>
+                <p class="govuk-body">
+                    Diversity managers should:
+                </p>
+                <ul class="govuk-list govuk-list--bullet">
+                    <li>Have a senior/executive role within the organisation</li>
+
+                    <li>Have visibility of internal data</li>
+
+                    <li>Be in the position to ask for more information on why decisions were made</li>
+
+                    <li>Be empowered to develop and implement diversity strategies and policies</li>
+
+                </ul>
+            </div>
+        </li>
+    </ol>
+</div>
+
+<div class="govuk-accordion" data-module="govuk-accordion" id="accordion-default">
+    <div class="govuk-accordion__section ">
+        <div class="govuk-accordion__section-header">
+            <h2 class="govuk-accordion__section-heading">
+                <span class="govuk-accordion__section-button" id="accordion-default-heading-1">
+                    References
+                </span>
+            </h2>
+        </div>
+        <div id="accordion-default-content-1" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-1">
+            @{
+                var referenceIndex = 1;
+            }
+            <div>
+                <h4 class="govuk-heading-s">@(referenceIndex++). Include multiple women in shortlists for recruitment and promotions</h4>

```

Comment:
Could you use the new `<ol>` / `<li>` structure for the References section on this page?